### ISSUE TITLE
Multiple usages of Broccoli-Eyeglass run the last compilation each time

### ISSUE THREAD

**Issue by alanhogan** [sass-eyeglass/broccoli-eyeglass#14]
Take a look at the example project: https://github.com/alanhogan/broccoli-eyeglass/tree/420519fc60a7d85db8d9fe6a98ea305073e06cdb/examples/example-4

Summary: If you have two or more instances of `BroccoliEyeglass`, they will all compile the same sass files as the *last* instance, and return the same output trees.

![screen shot of directory](https://cloud.githubusercontent.com/assets/6808/9240780/4aeeffd4-411f-11e5-8127-d69fbf94a118.png)

```js
/* Brocfile.js */

var BroccoliEyeglass = require('broccoli-eyeglass'),
  mergeTrees = require('broccoli-merge-trees'),
  fooOutput,
  barOutput;

fooOutput = new BroccoliEyeglass(['foo'], {
  cssDir: 'foo',
  verbose: true
});

console.log("fooOutput", fooOutput);

barOutput = new BroccoliEyeglass(['bar'], {
  cssDir: 'bar',
  verbose: true
});

console.log("barOutput", barOutput);

module.exports = mergeTrees([fooOutput, barOutput]);
```

**output**

```
examples/example-4[example-called-multiple-times]% broccoli build dist
fooOutput { '0': 'foo', relativeAssets: undefined }
barOutput { '0': 'bar', relativeAssets: undefined }
compile (0.005s) bar/bar.scss => bar/bar.css
compile (0.001s) bar/bar.scss => bar/bar.css
Error: Merge error: file bar/bar.css exists in /Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/tmp/broccoli_merge_trees-input_base_path-gZqVFhW6.tmp/0 and /Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/tmp/broccoli_merge_trees-input_base_path-gZqVFhW6.tmp/1
Pass option { overwrite: true } to mergeTrees in order to have the latter file win.
    at mergeRelativePath (/Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/node_modules/broccoli-merge-trees/index.js:118:19)
    at mergeRelativePath (/Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/node_modules/broccoli-merge-trees/index.js:142:15)
    at BroccoliMergeTrees.build (/Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/node_modules/broccoli-merge-trees/index.js:36:3)
    at /Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/read_compat.js:61:34
    at lib$rsvp$$internal$$tryCatch (/Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:505:17)
    at lib$rsvp$$internal$$publish (/Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:476:11)
    at lib$rsvp$asap$$flush (/Users/ahogan/Repos/broccoli-eyeglass/examples/example-4/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:1198:9)
    at process._tickCallback (node.js:355:11)

Build failed
```

**Comment by alanhogan** [comment]
I think I fixed this. Expect a PR shortly.

**Comment by alanhogan** [comment]
Fixed in #15.

This example now runs perfectly as expected. https://github.com/alanhogan/broccoli-eyeglass/tree/example-called-multiple-times/examples/example-4

**Comment by eoneill** [comment]
closing as fixed with #15. @chriseppstein we should publish a new version to npm
